java/src/main/java/com/codurance/training/tasks/service/Utility.java

```diff
@@ -0,0 +1,11 @@
+package com.codurance.training.tasks.service;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public interface Utility {
+	public Date parseDate(String deadline);
+	public String parseDateToString(Date date);
+	public boolean checkIdValidity(String id);
+}
```

Comment:
Utility class should not have interfaces
Ref: http://blog.vanillajava.blog/2011/06/best-practice-dont-create-instances-of.html